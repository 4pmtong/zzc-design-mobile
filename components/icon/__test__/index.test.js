import React from 'react';
import { mount, shallow, render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import Icon from '../index.jsx';

test( 'Button', () => {
  const wrapper = render( <div>
    <Icon type="success" />
    <Icon type="error" />
    <Icon type="waring" />
  </div> );
  expect( renderToJson( wrapper ) ).toMatchSnapshot();
} );