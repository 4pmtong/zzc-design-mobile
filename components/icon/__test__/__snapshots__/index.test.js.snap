// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<div>
  <svg
    class="zzc-icon zzc-icon-md zzc-icon-success"
  >
    <use
      href="#zzc-icon-success"
    />
  </svg>
  <svg
    class="zzc-icon zzc-icon-md zzc-icon-error"
  >
    <use
      href="#zzc-icon-error"
    />
  </svg>
  <svg
    class="zzc-icon zzc-icon-md zzc-icon-waring"
  >
    <use
      href="#zzc-icon-waring"
    />
  </svg>
</div>
`;
